package.json

```diff
@@ -15,6 +15,10 @@
     "clean": "lerna run clean --scope @shapeshiftoss/* && rm -rf coverage dist node_modules",
     "format": "prettier --check .",
     "format:fix": "prettier --write .",
+    "dev:ethereum:up": "COINSTACK=ethereum docker-compose up",
+    "dev:ethereum:down": "COINSTACK=ethereum docker-compose down",
+    "dev:bitcoin:up": "COINSTACK=bitcoin docker-compose up",
+    "dev:bitcoin:down": "COINSTACK=bitcoin docker-compose down",
```

Comment:
So we decided to remove these since there are going to be a lot of coin stacks and it doesn't provide the flexibility to spin up specific services.